### PR TITLE
Do not drop any physical slots on standby.

### PR DESCRIPTION
**Pull request by ashucoek**
At present, this tool synchronizes the logical slots from primary to standby. So it makes no sense to drop a physical slot on standby, regardless of whether it is present on the primary or not.

### CODE REVIEW

**Review by petere — APPROVED** [review]

**Comment by ashucoek** [comment]
Can you please merge the PR to the main branch. I don't have write access so I can't merge it.

**Pull request merged**
